File: runtime/paths.mjs
import path from "node:path";

export const SERVER = path.resolve(process.env.SERVER_DIR || "/server");
export const GAME = path.join(SERVER, "game/csgo");
export const STATE = path.resolve(process.env.STATE_DIR || "/state");
export const CONFIG = path.resolve(process.env.CONFIG_DIR || "/config");
export const STEAMCMD = path.resolve(process.env.STEAMCMD_DIR || "/opt/steamcmd");
export const SOCKET = process.env.CONTROL_SOCKET || "/run/cs2kz/control.sock";
export const PTERODACTYL = process.env.PTERODACTYL === "1";

File: runtime/paths.py
import os
from pathlib import Path

SERVER = Path(os.environ.get("SERVER_DIR", "/server"))
GAME = SERVER / "game/csgo"
STATE = Path(os.environ.get("STATE_DIR", "/state"))
CONFIG = Path(os.environ.get("CONFIG_DIR", "/config"))
STEAMCMD = Path(os.environ.get("STEAMCMD_DIR", "/opt/steamcmd"))
SOCKET = Path(os.environ.get("CONTROL_SOCKET", "/run/cs2kz/control.sock"))
PTERODACTYL = os.environ.get("PTERODACTYL") == "1"
